net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+			<description>
+				<![CDATA[
+					Set to 0 to disable resolving ip address to host names.<br/>
+					Set to 1 to enable resolving ip address to host names.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Net SNMP Perl Enable</fielddescr>
+			<fieldname>net_snmp_perl_enable</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to enable the use of the Perl module from the UCD-SNMP / NET-SNMP package.<br/>
+					This is required for $v variable substitution to work, and also for some other options<br/>
+					that are enabled in this .ini file.<br/>
+					Set to 0 to disable the use of the Perl module from the UCD-SNMP / NET-SNMP package.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Net SNMP Perl Cache Enable</fielddescr>
+			<fieldname>net_snmp_perl_cache_enable</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to enable caching of OID and ENUM translations when net_snmp_perl_enable is<br/>
+					enabled.  Enabling this should result in faster translations.<br/>
+					Set to 0 to disable caching.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Net SNMP Perl Best Guess</fielddescr>
+			<fieldname>net_snmp_perl_best_guess</fieldname>
+			<description>
+				<![CDATA[
+					This sets the best_guess parameter used by the UCD-SNMP / NET-SNMP Perl module for<br/>
+					translating symbolic nams to OIDs and vice versa.<br/>
+					For UCD-SNMP, and Net-SNMP 5.0.8 and previous versions, set this value to 0.<br/>
+					For Net-SNMP 5.0.9, or any Net-SNMP with patch 722075 applied, set this value to 2.<br/>
+					A value of 2 is equivalent to -IR on Net-SNMP command line utilities.<br/>
+					UCD-SNMP and Net-SNMP 5.0.8 and previous may not be able to translate certain formats of<br/>
+					symbolic names such as RFC1213-MIB::sysDescr. Net-SNMP 5.0.9 or patch 722075 will allow<br/>
+					all possibilities to be translated.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Log Trap OID</fielddescr>
+			<fieldname>translate_log_trap_oid</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to use the default of numerical OID<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.